Keypad/Keypad.module.less

```diff
@@ -111,11 +149,72 @@
 						color: @agate-keypad-button-active-color;
 					}
 				}
+
+				.focus({
+					.client {
+						color: @agate-keypad-button-focus-color;
+					}
+				});
+			});
+		}
+
+		// 'accept' and 'decline' classes are set for design requirements accuracy
+		&.accept,
+		&.decline {
+			.focus({
+				.client {
+					color: @agate-keypad-button-focus-color;
+				}
+			});
+
+			.active({
+				.focus({
+					.client {
+						color: @agate-keypad-button-focus-color;
```

Comment:
is there no way to escape from a lot of color override, right? :(

Reply:
I assume it is temporary. At least from line 162 to 198, as we don't have colored icons yet. 
Also, there are many overrides so that both Silicon and Gallium look good either on day or on night mode.  
I suppose we can refactor it when design requirements are complete.